profiles/package.mask

```diff
@@ -32,6 +32,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Jakov Smolic <jakov.smolic@sartura.hr> (2021-02-01)
+# Fails to build with gcc-10, no maintainer, last release in 2004,
+# no reverse dependencies.
+# Removal on 2021-03-03.  Bug #707504.
```

Comment:
prefer to always write `Removal in 30 days`

Reply:
Sure, I'm using mgorny's script, it switched to `Removal on date` recently, but I'll update it

Reply:
Done